my-webxr-app/src/components/DataPointMenu.tsx

```diff
@@ -18,7 +18,7 @@ export default function DataPointMenu(
   return (
     <Billboard visible={selectedDataPoint != null} {...billboardProps}>
       <Plane args={[1.25, 0.8]}>
-        <Text fontSize={0.075} color="black">
+        <Text fontSize={0.075} color="black" depthOffset={-4}>
```

Comment:
is the smaller the offset, the closer the text is to the user?
is there any particular reason for value -4? or does it just work?

Reply:
btw, good catch on the depthOffset! :))

Reply:
> is the smaller the offset, the closer the text is to the user? is there any particular reason for value -4? or does it just work?

I'm wondering the same...

Reply:
1. Yes, the smaller the number the closer it is to the observer. I added a comment to record that.
2. I chose -4 because I slowly decreased it until everything looked steady. in the 0 to -3 range, there were still a few viewing angles that flickered.